fonts/fonts.css

```diff
@@ -0,0 +1,26 @@
+@font-face {
+  font-family: 'Montserrat';
+  src: 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/LOCO__Oxana/pullRequest?prid=5565108)